### DISCUSSION TITLE
搭建好服务后，本机使用webdav的速度很快，但是局域网的其他设备速度很慢。

### DISCUSSION THREAD

**Discussion by Brainiac5000** [AlistGo/alist#2647] [Q&A]
环境：
所有设备全走的5g wifi，两台电脑无线局域网共享速度是50m/s（由于无线上传下载带宽问题）。如果全使用有线可达100m/s。

使用alist搭建好服务后，添加了115和阿里云盘，使用webdav映射在本机上的下载速度很快，115能达到20m/s，阿里云能到达40m/s。
但是局域网的其他设备速度很慢。115只能跑到8m/s，阿里云可以到18m/s。看起来速度是折半了。感觉是不是那个位置的参数没有设置正确，所以特来求助。

**Comment by xinxinenjoy** [comment]
现在还没解决局域网设备访问的问题，请问您是怎么操作的？
我本机可以正常访问，可局域网内设备就是无法连接

**Reply by Brainiac5000** [reply]
webdav协议 5244端口 /dav路径 直接挂载就行。
